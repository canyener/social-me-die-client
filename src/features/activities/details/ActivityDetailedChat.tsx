import React, { Fragment } from 'react'
import { Segment, Header, Form, Button, Comment } from 'semantic-ui-react'

const ActivityDetailedChat = () => {
  return (
    <Fragment>
      <Segment
        textAlign='center'
        attached='top'
        inverted
        color='teal'
        style={{ border: 'none' }}
      >
        <Header>Chat about this event</Header>
      </Segment>
      <Segment attached>
        <Comment.Group>
          <Comment>
            <Comment.Avatar src='/assets/user.png' />
            <Comment.Content>
              <Comment.Author as='a'>Can</Comment.Author>
              <Comment.Metadata>
                <div>Today at 5:42PM</div>
              </Comment.Metadata>
              <Comment.Text>Niceee!</Comment.Text>
              <Comment.Actions>
                <Comment.Action>Reply</Comment.Action>
              </Comment.Actions>
            </Comment.Content>
          </Comment>

          <Comment>
            <Comment.Avatar src='/assets/user.png' />
            <Comment.Content>
              <Comment.Author as='a'>Onur the Designer</Comment.Author>
              <Comment.Metadata>
                <div>5 days ago</div>
              </Comment.Metadata>
              <Comment.Text>Dude, this is awesome. Thanks so much</Comment.Text>
              <Comment.Actions>
                <Comment.Action>Reply</Comment.Action>
              </Comment.Actions>
            </Comment.Content>
          </Comment>

          <Form reply>
            <Form.TextArea />
            <Button
              content='Add Reply'
              labelPosition='left'
              icon='edit'
              primary
            />
          </Form>
        </Comment.Group>
      </Segment>
    </Fragment>
  )
}

export default ActivityDetailedChat