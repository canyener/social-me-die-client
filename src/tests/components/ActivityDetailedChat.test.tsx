import {shallow } from 'enzyme'
import ActivityDetailedChat from '../../features/activities/details/ActivityDetailedChat'

test('Should render ActivityDetailedChat correctly', () => {
  const wrapper = shallow(<ActivityDetailedChat />)

  expect(wrapper).toMatchSnapshot()
})