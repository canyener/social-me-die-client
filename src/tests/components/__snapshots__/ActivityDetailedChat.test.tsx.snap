// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ActivityDetailedChat correctly 1`] = `
<Fragment>
  <Segment
    attached="top"
    color="teal"
    inverted={true}
    style={
      Object {
        "border": "none",
      }
    }
    textAlign="center"
  >
    <Header>
      Chat about this event
    </Header>
  </Segment>
  <Segment
    attached={true}
  >
    <CommentGroup>
      <Comment>
        <CommentAvatar
          src="/assets/user.png"
        />
        <CommentContent>
          <CommentAuthor
            as="a"
          >
            Can
          </CommentAuthor>
          <CommentMetadata>
            <div>
              Today at 5:42PM
            </div>
          </CommentMetadata>
          <CommentText>
            Niceee!
          </CommentText>
          <CommentActions>
            <CommentAction
              as="a"
            >
              Reply
            </CommentAction>
          </CommentActions>
        </CommentContent>
      </Comment>
      <Comment>
        <CommentAvatar
          src="/assets/user.png"
        />
        <CommentContent>
          <CommentAuthor
            as="a"
          >
            Onur the Designer
          </CommentAuthor>
          <CommentMetadata>
            <div>
              5 days ago
            </div>
          </CommentMetadata>
          <CommentText>
            Dude, this is awesome. Thanks so much
          </CommentText>
          <CommentActions>
            <CommentAction
              as="a"
            >
              Reply
            </CommentAction>
          </CommentActions>
        </CommentContent>
      </Comment>
      <Form
        as="form"
        reply={true}
      >
        <FormTextArea
          as={[Function]}
          control={[Function]}
        />
        <Button
          as="button"
          content="Add Reply"
          icon="edit"
          labelPosition="left"
          primary={true}
        />
      </Form>
    </CommentGroup>
  </Segment>
</Fragment>
`;
